compress/README.md

```diff
@@ -0,0 +1,124 @@
+                                                                       u'minimumRamGb': 3.75}},
+                               u'pipelineArgs': { u'clientId': u'',
+                                                  u'inputs': { u'inputFile': u'gs://genomics-public-data/platinum-genomes/vcf/NA12877_S1.genome.vcf'},
+                                                  u'logging': { u'gcsPath': u'gs://YOUR-BUCKET/compress/logging_path'},
+                                                  u'outputs': { u'outputFile': u'gs://YOUR-BUCKET/compress/output_path/NA12877_S1.genome.vcf.gz'},
+                                                  u'projectId': u'YOUR-PROJECT-ID',
+                                                  u'resources': { u'disks': [ { u'autoDelete': False,
+                                                                                u'mountPoint': u'',
+                                                                                u'name': u'data',
+                                                                                u'readOnly': False,
+                                                                                u'sizeGb': 200,
+                                                                                u'source': u'',
+                                                                                u'type': u'PERSISTENT_HDD'}],
+                                                                  u'minimumCpuCores': 0,
+                                                                  u'minimumRamGb': 1,
+                                                                  u'preemptible': False,
+                                                                  u'zones': [ u'us-central1-a',
+                                                                              u'us-central1-b',
+                                                                              u'us-central1-c',
+                                                                              u'us-central1-f',
+                                                                              u'us-east1-b',
+                                                                              u'us-east1-c',
+                                                                              u'us-east1-d']},
+                                                  u'serviceAccount': { u'email': u'default',
+                                                                       u'scopes': [ u'https://www.googleapis.com/auth/compute',
+                                                                                    u'https://www.googleapis.com/auth/devstorage.full_control',
+                                                                                    u'https://www.googleapis.com/auth/genomics']}}},
+                 u'startTime': u'2016-03-10T04:53:57.000Z'},
+  u'name': u'operations/YOUR-NEW-OPERATION-ID'}
+
+```
+
+## (2) Check the results
```

Comment:
It may be time to think about moving these tutorials to RTD in a future PR for easier maintenance and consistency across tutorials.

For example, this "check the results" block could be an include file.